hc/front/views.py

```diff
@@ -552,3 +552,36 @@ def privacy(request):
 
 def terms(request):
     return render(request, "front/terms.html", {})
+
+
+# Failed Jobs
+@login_required
+def failed_jobs(request):
+    q = Check.objects.filter(user=request.team.user).order_by("created")
```

Comment:
Are we able to filter with the status here? If so, it looks better to retrieve downed checks from the database.